examples/subscriber.rs

```diff
@@ -124,7 +124,7 @@ async fn main() {
                 ),
             }
 
-            packet_stats.count(1);
+            packet_stats.log_metrics(1, publication.payload.len());
```

Comment:
Only counting the payload will undercount the receive rate. Eg for 32-byte payloads subscriber will report 32/39 == 82% of what fast_publisher reports.

You could do:

```rust
            packet_stats.count(
                1,
                1 + // packet type
                1 + // remaining length
                2 + // topic length prefix
                publication.topic_name.len() +
                publication.payload.len(),
            );
```

... but this still assumes remaining length is encoded as one byte, which is not true for say 8192.

If you really need bytes, the better place for these metrics is in the `LoggingStream` and `LoggingSink`, since they know both how many messages and how many bytes they processed.

Reply:
Ahh gotcha, did a cursory overview... let me update with using real value from either the stream or sink!

Reply:
Sorry, just noticed I misspoke. I meant `IoStream` and `IoSink` (in `examples/transport/tokio.rs`), not `Logging*` (which are just wrappers for existing stream / sink).

Reply:
Ahh okay that makes more sense, I was looking at LoggingStream and was scratching my head on why here?